test_data/yam/agama_rmt.yaml

```diff
@@ -1,6 +1,14 @@
 ---
 repos:
-  - name: SLE-Product-SLES-%VERSION%
-    alias: SUSE_Linux_Enterprise_Server_%VERSION%_%ARCH%:SLE-Product-SLES-%VERSION%
+  - name: SLE-Product-SLES-16.0
+    alias: SUSE_Linux_Enterprise_Server_16.0:SLE-Product-SLES-16.0
```

Comment:
For now, due to the different VERSION for production and development group, I suggest to use static '16.0' instead of %VERSION%, reserve ARCH.  ex: SUSE_Linux_Enterprise_Server_16.0_%ARCH%:SLE-Product-SLES-16.0